kevm_pyk/src/kevm_pyk/solc_to_k.py

```diff
@@ -259,7 +265,10 @@ def gen_claims_for_contract(empty_config: KInner, contract_name: str, calldata_c
         'ORIGIN_CELL': KVariable('ORIGIN_ID'),
         'ID_CELL': Foundry.address_TEST_CONTRACT(),
         'CALLER_CELL': KVariable('CALLER_ID'),
-        'LOCALMEM_CELL': KApply('.Memory_EVM-TYPES_Memory'),
+        'ACCESSEDSTORAGE_CELL': KApply('.Map'),
+        'ACTIVEACCOUNTS_CELL': KApply('_Set_',[ KApply('_Set_',[ KApply('SetItem',[Foundry.address_TEST_CONTRACT()]), KApply('SetItem',[Foundry.address_CHEATCODE()])]), KApply('_Set_',[ KApply('SetItem',[Foundry.address_CALLER()]), KApply('SetItem',[Foundry.address_HARDHAT_CONSOLE()])])]),
```

Comment:
This should be generated using something like
```python
build_assoc(KApply('_Set_'), KLabel('SetItem'), [intToken(0), intToken(100), intToken(2), intToken(3)])
```
but I couldn't get it right so far.

Reply:
It should be something like this:

```
build_assoc(KApply('.Set'), KLabel('_Set_'), map(KLabel('SetItem'), [intToken(0), intToken(100), ...]))
```